packages/token/src/erc20/extensions/erc4626/erc4626.cairo

```diff
@@ -230,18 +249,48 @@ pub mod ERC4626Component {
     /// See the example:
     /// https://github.com/OpenZeppelin/cairo-contracts/tree/main/packages/test_common/src/mocks/erc4626.cairo
     pub trait ERC4626HooksTrait<TContractState, +HasComponent<TContractState>> {
-        /// Hooks into `InternalImpl::_withdraw`.
-        /// Executes logic before burning shares and transferring assets.
-        fn before_withdraw(ref self: ComponentState<TContractState>, assets: u256, shares: u256) {}
-        /// Hooks into `InternalImpl::_withdraw`.
-        /// Executes logic after burning shares and transferring assets.
-        fn after_withdraw(ref self: ComponentState<TContractState>, assets: u256, shares: u256) {}
         /// Hooks into `InternalImpl::_deposit`.
         /// Executes logic before transferring assets and minting shares.
```

Comment:
I think we should elaborate on this comments as well, so is easier for users to understand the flow.

I would explain that:
- The fee is comming from the corresponding function in the fee config trait, and the assets and shares are already adjusted from that fee when they reach the hook.